### PR TITLE
Musicxml import logging

### PR DESCRIPTION
**Pull request by lvinken**
MusicXML import refactoring: moved some common code between pass 1 and 2 to a separate class.

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
testutils don't build and therefore the mtests fail

**Comment by lvinken** [comment]
Thanks, should be fixed now

**Comment by lasconic** [comment]
I will try the merge with squash on this PR.

**Comment by lasconic** [comment]
It doesn't look too bad ! 
https://github.com/musescore/MuseScore/commits/master

Regarding the PR, good to have separate logger for musicxml import!

File: mscore/importmxmllogger.h [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#define __IMPORTMXMLLOGGER_H__
+
+class QXmlStreamReader;
+
+namespace Ms {
+
+class MxmlLogger {
+public:
+      enum class Level : char {
+            TRACE, INFO, ERROR
```
Comment (Jojo-Schmitz) [inline mscore/importmxmllogger.h]: INFO and ERROR conflicts with something in a Windows specific header, so it doesn't build there, renaming to XINFO and XERROR fixes the build problem

**Comment by lasconic** [comment]
Maybe this whole logging system could be replaced by https://blog.qt.io/blog/2014/03/11/qt-weekly-1-categorized-logging/ ?

**Comment by lvinken** [comment]
Actually in my opinion we could use both the qC* logging (selectively enable logging to debug user issues) and reporting of the import result in the GUI instead of as console output. When designing my logging facility I was targeting the second case. As all of the importer's output now goes through a dedicated class, it is easy to change where it goes to.

**Pull request merged**